scripts/preview_docc.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftStatsdClient open source project
+##
+## Copyright (c) 2022 the SwiftStatsdClient project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftStatsdClient project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the Swift Distributed Actors open source project
+##
+## Copyright (c) 2018-2019 Apple Inc. and the Swift Distributed Actors project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.md for the list of Swift Distributed Actors project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+xcrun swift package --disable-sandbox preview-documentation --target $1
```

Comment:
I dont think `xcrun` is required, and it is not available on linux. what was the reason of using it here?

cc @ktoso